lintro/ai/providers/cursor.py

```
@@ -0,0 +1,259 @@
+    ) -> AIResponse:
+        """Run a completion via ``agent --print``.
+
+        The system prompt is prepended to the user prompt since the
+        CLI does not have a dedicated system-prompt flag.
+        """
+        effective_max = min(max_tokens, self._max_tokens)
+        combined_prompt = prompt
+        if system:
+            combined_prompt = f"{system}\n\n---\n\n{prompt}"
+        combined_prompt = (
+            f"{combined_prompt}\n\n" f"[Respond in at most {effective_max} tokens.]"
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)